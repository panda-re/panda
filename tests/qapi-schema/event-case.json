# TODO: might be nice to enforce naming conventions; but until then this works
# even though events should usually be ALL_CAPS
{ 'event': 'oops' }
