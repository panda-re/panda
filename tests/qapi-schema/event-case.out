enum QType ['none', 'qnull', 'qint', 'qstring', 'qdict', 'qlist', 'qfloat', 'qbool']
    prefix QTYPE
event oops None
   boxed=False
object q_empty
